manim/utils/tex.py

```diff
@@ -12,7 +12,7 @@
 
 
 class TexTemplate:
-    """TeX templates are used for creating Tex() and MathTex() objects.
+    r"""TeX templates are used for creating Tex() and MathTex() objects.
```

Comment:
```suggestion
    """TeX templates are used for creating Tex() and MathTex() objects.
```
Does it make a difference if it is a raw string?

Reply:
Raw stops backslashes from breaking things